mainui/settings/persistence.md

```diff
@@ -9,21 +9,26 @@ title: Settings - Persistence
 **Persistence** stores data over time.
 The data may be retrieved at a later time, for example to restore your system after startup, or to prepare graphs for display on the UI.
 
-openHAB ships with the [RRD4J persistence service](/addons/persistence/rrd4j/) installed by default, which persists every Item on every state change and at least once a minute.
-Additionally, it restores the last stored value at system startup.
+openHAB ships with the [RRD4J persistence service](/addons/persistence/rrd4j/), which by default:
 
-Learn more about persistence in the [persistence docs]({{base}}/configuration/persistence.html).
+- persists every Item on every state change, and at least once a minute.
```

Comment:
I worry about reproducing the default strategy here. If rrd4j changes it whom ever changes it there has to know to change it here too.